cmd/clusterawsadm/cloudformation/bootstrap/template_test.go

```diff
@@ -184,7 +184,6 @@ func Test_RenderCloudformation(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			os.WriteFile("/tmp/tmp1", tData, 0600)
```

Comment:
This line was not testing anything, just writing content in file which is not checked.